packages/browser-crawler/src/internals/browser-crawler.ts

```diff
@@ -260,6 +260,12 @@ export interface BrowserCrawlerOptions<
      * By default, they are expanded automatically. Use this option to disable this behavior.
      */
     ignoreShadowRoots?: boolean;
+
+    /**
+     * Whether to ignore `iframes` when processing the page content via `parseWithCheerio` helper.
+     * By default, `iframes` are expanded automatically. Use this option to disable this behavior.
+     */
+    ignoreIframes?: boolean;
```

Comment:
Since this is also a parameter to the `parseWithCheerio` helper itself, do we really need this?

Reply:
That's the mess-up, the `parseWithCheerio` util function accepts the `page` and `ignore(Iframes|ShadowRoots)` params, but the context helper available from `requestHandler` accepts the new `waitForSelector` + `timeout` args.

It's a mess, but the proposed changes are IMO the most consistent approach.

cc @B4nan - as Honza is OOO and you have initiated most of the related changes anyway - what do you think?

Reply:
I would keep it as is. We can rethink this in v4, I guess an options parameter would do it (and we can implement the support in v3 conditionally in the end).

Reply:
agreed, v4 debt it is 👍🏽